Refresh images to reflect new UX (#3137)
harness/developer-hub: docs/infra-as-code-management/workspaces/provision-workspace.md

4. Enter a stage name and an optional description and tag. Select **Set Up Stage**. 

> 5. On the **Workspace** tab, select an existing workspace or select **+ Create New Workspace** to create a new Workspace. A workspace can also be determined during execution if the value is set to be runtime input

Reviewer: ```suggestion
5. On the **Workspace** tab, select an existing workspace or select **+ Create New Workspace** to create a new workspace. A workspace can also be determined during execution if the value is set to be `runtime input`.
```

> 6. Select the **Execution** tab, and then select "Provision" from 'Common Operations'.

Reviewer: ```suggestion
6. Select the **Execution** tab, and then from **Common Operations**, select **Provision**.
```

> 7. Click **Use Strategy**.

Reviewer: ```suggestion
7. Select **Use Strategy**.
```

## Destory a workspace

You can use the Harness Visual Editor or YAML to create a pipeline that will tear down (destroy) a workspace. 

```mdx-code-block
<Tabs>
  <TabItem value="Visual" label="Visual">
```

> 1. Follow Steps 1-5 as described in the "Provision a Workspace" section

Reviewer: Don't make users go hunt for steps. Provide the steps here.

> 2. Select the **Execution** tab and select "Destroy" from 'Common Operations'.

Reviewer: ```suggestion
2. On the **Execution** tab, in **Common Operations**, select **Destroy**.
```

> 3. Click **Use Strategy**.

Reviewer: ```suggestion
3. Select **Use Strategy**.
```
